site/hxapi/asynchronous-bookings.md

```diff
@@ -0,0 +1,169 @@
+|-------------------------|-----------|-----------------|------------|--------------------------------------------------------------------|
+| includeAsyncProducts    | Boolean   | `true`/`false` | N          | If `true`, the booking is treated as asynchronous. Default is `false`. |
+
+### Behavior
+
+- **`true`:** The booking will be processed asynchronously. A response will indicate that the booking has been created but not yet confirmed.
+- **`false`:** The booking will follow the standard synchronous process.
+
+### Request
+Use the existing request examples: [/hxapi/parking/bkg/](/hxapi/parking/bkg/) and add the `includeAsyncProducts=true` query parameter.
+
+### Response Structure for Async Bookings
+
+When the `includeAsyncProducts` parameter is set to `true` and the booking is processed asynchronously, the response will closely resemble the normal synchronous booking response. However, certain properties will be missing, as they are only available after the booking is fully confirmed.
+
+#### Missing Properties:
+- **QR Code:** Not provided until the booking is confirmed.
+- **Supplier Reference:** Only included in the response once the supplier confirms the booking.
+- **Joining Instructions:** These details will be added once the booking reaches a `CONFIRMED` state.
+
+#### Added Property:
+- **`bookingStatus`:** Indicates the current status of the booking (`PENDING`, `CONFIRMED`, or `FAILED`).
+- **`suggestedPollingInterval`:** Specifies the recommended interval in seconds for polling the view booking endpoint. The polling frequency varies based on the product type and supplier.
+
+---
+
+## Booking Polling  
+
+When working with asynchronous bookings, partners must poll our API to determine the status of a booking and when full booking details are ready.  
+
+### When Polling is Required  
+
+Polling is required when a booking has been submitted with the `includeAsyncProducts=true` parameter.
+The `bookingStatus` field in the response from the `view booking` endpoint indicates the current status of a booking.  
+
+Booking details such as the QR code, supplier reference, and other information will only be provided once the `bookingStatus` field is in a `CONFIRMED` state.
+
+### How Often to Poll  
+
+The frequency of polling depends on the product type and supplier.
+On the initial booking response, we provide a `suggestedPollingInterval` field to guide partners on the recommended polling frequency.
+
+### Booking Status Values  
+
+| Status       | Description                                                                 |
+|--------------|-----------------------------------------------------------------------------|
+| `PENDING`    | Booking has been submitted but not yet confirmed with the supplier.         |
+| `CONFIRMED`  | Booking has been successfully confirmed by the supplier. Full details are available. |
+| `FAILED`     | Booking has failed with the supplier. |
+
+---
+
+## API Endpoint
```

Comment:
Can you adjust these heading levels?
`Booking Polling` is a level 2, and this should be under that.

Reply:
adjusted